partials/socials.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="socials">
+<div class="socials row">
   {{#each socials}}
-    <a href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
+    <a class="col-xs-4 col-sm-2 col-md-2" href="{{this}}" target="_blank"><i class="fa fa-{{@key}}"></i></a>
```

Comment:
I was trying to avoid this in purpose because if we add new social icons it will break the layout